src/UhdmAst.h

```diff
@@ -29,6 +30,9 @@ struct UhdmShared {
     std::map<std::string, AstNodeModule*> top_nodes;
 };
 
+// Parse vpi*Typespec nodes
+AstNode* process_typespec(vpiHandle obj_h, UhdmShared& shared);
```

Comment:
Why add this? Do you plan to use it somewhere else?

Reply:
I needed the declaration before definition, because that function and `getDType` use each other.